lib/node_modules/@stdlib/blas/ext/base/dapxsumkbn/src/addon.c

```diff
@@ -36,14 +37,36 @@ static napi_value addon( napi_env env, napi_callback_info info ) {
 	STDLIB_NAPI_ARGV( env, info, argv, argc, 4 );
 	STDLIB_NAPI_ARGV_INT64( env, N, argv, 0 );
 	STDLIB_NAPI_ARGV_DOUBLE( env, alpha, argv, 1 );
-	STDLIB_NAPI_ARGV_INT64( env, stride, argv, 3 );
-	STDLIB_NAPI_ARGV_STRIDED_FLOAT64ARRAY( env, X, N, stride, argv, 2 );
+	STDLIB_NAPI_ARGV_INT64( env, strideX, argv, 3 );
+	STDLIB_NAPI_ARGV_STRIDED_FLOAT64ARRAY( env, X, N, strideX, argv, 2 );
 
 	napi_value v;
-	napi_status status = napi_create_double( env, stdlib_strided_dapxsumkbn( N, alpha, X, stride ), &v );
+	napi_status status = napi_create_double( env, API_SUFFIX(stdlib_strided_dapxsumkbn)( N, alpha, X, strideX ), &v );
 	assert( status == napi_ok );
 
 	return v;
 }
 
-STDLIB_NAPI_MODULE_EXPORT_FCN( addon )
+/**
+* Receives JavaScript callback invocation data.
+*
+* @param env    environment under which the function is invoked
+* @param info   callback data
+* @return       Node-API value
+*/
+static napi_value addon_method( napi_env env, napi_callback_info info ) {
+	STDLIB_NAPI_ARGV( env, info, argv, argc, 5 );
+	STDLIB_NAPI_ARGV_INT64( env, N, argv, 0 );
+	STDLIB_NAPI_ARGV_DOUBLE( env, alpha, argv, 1 );
+	STDLIB_NAPI_ARGV_INT64( env, strideX, argv, 3 );
+	STDLIB_NAPI_ARGV_INT64( env, offsetX, argv, 4 );
+	STDLIB_NAPI_ARGV_STRIDED_FLOAT64ARRAY( env, X, N, strideX, argv, 2 );
+
+	napi_value v;
+	napi_status status = napi_create_double( env, API_SUFFIX(stdlib_strided_dapxsumkbn_ndarray)( N, alpha, X, strideX, offsetX ), &v );
+	assert( status == napi_ok );
```

Comment:
@headlessNode Let's use the `STDLIB_NAPI_CREATE_DOUBLE` macro, rather than manually create the double here. See `@stdlib/blas/base/ddot` for an example.